groot/cleaner.go

```diff
@@ -1,7 +1,9 @@
 package groot
 
 import (
+	"fmt"
 	"time"
+	"fmt"
```

Comment:
Double `fmt` here. Could you remove one?

Reply:
Removing it right away.